<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>562</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Светофор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#5c5263;
font-family: sans-serif;
font-width: 500;
color: white;
font-size:30px;
}
QGroupBox#svetgroup{
background-color:#C0C0C0;
border-radius:15px;
}
QPushButton{
background-color:gray;
border-radius:30px;
color: white;
font-family: sans-serif;
font-width: 500;
color: white;
font-size:30px;
}
QPushButton#on_btn{
background-color:#C0C0C0;
border-radius:15px;
}
QPushButton#off_btn{
background-color:#C0C0C0;
border-radius:15px;
}
QLabel{
font-size:30px;
color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="svetgroup">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>100</y>
      <width>121</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string> </string>
    </property>
    <widget class="QPushButton" name="btn1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="off_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Выключить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="on_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Светофор</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
